aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -109,13 +109,13 @@ spec aptos_framework::staking_contract {
     }
 
     spec update_commision (staker: &signer, operator: address, new_commission_percentage: u64) {
-        // TODO: Call `distribute_internal`
+        // TODO: Call `distribute_internal` and Could not verify `update_distribution_pool`.
```

Comment:
"and Could" --> "and could".
Could you also fix the same typos that repeat below?